src/handlers/payout/index.ts

```diff
@@ -50,12 +50,13 @@ export const handleIssueClosed = async () => {
   }
 
   if (issue.state_reason === StateReason.COMPLETED) {
-    const payoutUrl = await generatePermit2Signature(recipient, priceInEth);
+    const payoutUrl = await generatePermit2Signature(recipient, priceInEth, issue.node_id);
     const tokenSymbol = await getTokenSymbol(paymentToken, rpc);
     const shortenRecipient = shortenEthAddress(recipient, `[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]`.length);
     logger.info(`Posting a payout url to the issue, url: ${payoutUrl}`);
     const comment = `### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
     await addCommentToIssue(comment, issue.number);
     await deleteLabel(issueDetailed.priceLabel);
+    await addLabelToIssue("Paid");
```

Comment:
Why was this added @Steveantor

Reply:
the paid label indicates that the bounty bot posted the permit link,
it makes the process somewhat faster instead of parsing through all the messages.